integration/hurl/tests_failed/assert_base64.hurl

```diff
@@ -9,5 +9,5 @@
 
 GET http://localhost:8000/assert-base64
 HTTP 200
-base64,bGluZTEKbGluZTIKbGluZTMK;
+base64, bGluZTEKbGluZTIKbGluZTMK;
```

Comment:
Same as previous, I don't see the need to change the Hurl file (maybe I'm missing something?). If there is no goog reason, better not to modify it

Reply:
I will revert but those tests are really confusing, locally are failing and as I already written `python3 integration.py` are failing in some way, and /bin/coverage.sh` in other way

Reply:
I'll try to test your PR when I've some time, it's surprising indeed... 

FYI, we´ll update the CI checks because contributors only needs to run `integration.py` and not `coverage`. Actually `coverage` is run before integration tests and this is misleading indeed.